Ship_Game/DeepSpaceBuildingWindow.cs

```diff
@@ -226,7 +290,19 @@ public override void Draw(SpriteBatch batch, DrawTimes elapsed)
                     }
                 }
 
-                batch.Draw(platform, cursorPos, new Color(0, 255, 0, 100), 0f, IconOrigin, (float)scale, SpriteEffects.None, 1f);
+                foreach (SolarSystem system in systems)
+                {
+                    if (system.Position.Distance(cursorWorldPos) <= system.Radius * 0.8f)
+                    {
+                        TargetSystem = system;
+                        break;
+                    }
+
+                    TargetSystem = null;
+                }

```

Comment:
This is really bad code, we already have a function in UniverseState and UniverseScreen for picking closest system to cursor world pos.